packages/contracts-bedrock/test/kontrol/proofs/interfaces/KontrolInterfaces.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Types } from "src/libraries/Types.sol";
+
+interface IOptimismPortal {
+    function GUARDIAN() external view returns (address);
+
+    function guardian() external view returns (address);
+
+    function paused() external view returns (bool paused_);
+
+    function proveWithdrawalTransaction(
+        Types.WithdrawalTransaction memory _tx,
+        uint256 _l2OutputIndex,
+        Types.OutputRootProof calldata _outputRootProof,
+        bytes[] calldata _withdrawalProof
+    )
+        external;
+
+    function finalizeWithdrawalTransaction(Types.WithdrawalTransaction memory _tx) external;
+}
+
+interface ISuperchainConfig {
+    function guardian() external view returns (address);
+
+    function paused() external view returns (bool paused_);
+
+    function pause(string memory _identifier) external;
```

Comment:
As noted in the proofs, this interface is out of date. Perhaps these should be auto-generated (or else validated in CI) using `cast interface`.

Reply:
The signature is still `function pause(string memory _identifier) external` on the latest develop, so resolving this https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/SuperchainConfig.sol#L68